docs/build.md

```diff
@@ -104,8 +104,14 @@ Learn more on <https://apt.llvm.org>
 
 ### Python Dependencies
 
+**macOS:**
 ```bash
-pip intall -r utils/python_requirements.txt
+pip3 install -r utils/python_requirements.txt
```

Comment:
I would prefer simple to add comment that we require _python>=3.5_ for a successful build.